service/history/workflow/update/registry.go

```diff
@@ -306,6 +306,7 @@ func (r *registry) admit(ctx context.Context) error {
 	if len(r.updates) >= r.maxInFlight() {
 		return serviceerror.NewResourceExhausted(
 			enumspb.RESOURCE_EXHAUSTED_CAUSE_CONCURRENT_LIMIT,
+			enumspb.RESOURCE_SCOPE_SYSTEM,
```

Comment:
FYI This limit is per-namespace as the registry is for a specific namespace; it comes from `WorkflowExecutionMaxInFlightUpdates` which is a per-namespace dynamic config value. 

@yycptt this sounds like a SCOPE_NAMESPACE issue to me but another opinion would be good